### DISCUSSION TITLE
Importing Information Model based on Companion Specification Machine Vision Part 2

### DISCUSSION THREAD

**Discussion by EricWeinz** [FreeOpcUa/opcua-asyncio#1342] [Q&A]
Hello everyone,

I'm trying to import a Companion Specification Machine Vision Part 2 (release candidate). For this I used Siemens SiOME and UA Modeler (I tried both) to create an information model ("my_information_model.xml"). When I import them into my Code I get the error message: 

Model is missing: http://opcfoundation.org/UA/ - Version: 1.05.02 - PublicationDate: 2022-11-01T00:00:00Z or newer
Model is missing: http://opcfoundation.org/UA/Machinery/ - Version: 1.03.0 - PublicationDate: 2023-08-01T00:00:00Z or newer
    raise ValueError("Server doesn't satisfy required XML-Models. Import them first!")
ValueError: Server doesn't satisfy required XML-Models. Import them first!

My imports:
    await server.import_xml(".../UA-Nodeset/DI/Opc.Ua.Di.NodeSet2.xml")
    await server.import_xml(".../UA-Nodeset/Machinery/Opc.Ua.Machinery.NodeSet2.xml")
    await server.import_xml(".../UA-Nodeset/Opc.Ua.MachineVision.AMCM.xml")
    await server.import_xml(".../UA-Nodeset/my_information_model.xml")

Do you know, what I'm doing wrong?
Thank you and have a nice day

**Comment by schroeder-** [comment]
We curently use http://opcfoundation.org/UA/ 1.05.01 Version. The easiest way is to edit my_information_model.xml and change the value from 1.05.02 -> 1.05.01 and the PublicationDate to 2022-02-24T00:00:00Z.

**Reply by EricWeinz** [reply]
Thank you for your answer. The message with "http://opcfoundation.org/UA/ " is gone now but the message for Machinery is still there.
Model is missing: http://opcfoundation.org/UA/Machinery/ - Version: 1.03.0 - PublicationDate: 2023-08-01T00:00:00Z

**Reply by schroeder-** [reply]
Did you download the lastest version of  http://opcfoundation.org/UA/Machinery/ ?

**Reply by EricWeinz** [reply]
It works! I downloaded the latest Machinery Version and edit the http://opcfoundation.org/UA/ 1.05.01 Version again in that .xml
Thank you!
